recipes/mapd-core-cpu/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "mapd-core-cpu" %}
+{% set version = "4.5.0" %}
+{% set sha256 = "d2a0f7ffd4366a38aed2a5a81e69afde5293aa03fdcbf5142ff2caed6cccce5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mapd/mapd-core/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - go
+    - make
+    - maven
+    - openjdk 8.*
+  host:
+    - arrow-cpp >=0.12
+    - bisonpp
+    - blosc
+    - boost-cpp 1.68.*
+    - boost 1.68.*
+    - clangdev 7.*
```

Comment:
Hmm it is weird that you are pinning clangdev and llvmdev. These should be handled by the `{{ compiler('cpp') }}`, and also do you really need clangdev on linux?

Reply:
the original recipe @pearu made pinned to 0.6 ... for gpu build (is not this case) it need 0.7 ... I changed here to 0.7 to see if it could fix some issues (it didn't happen). @pearu can provide more info about pinning clangdev/llvmdev and also about why it is needed in linux.

Reply:
in the notes made by @pearu (https://github.com/Quansight/pearu-sandbox/blob/master/README-omnisci.md#prepare-conda-environment-with-mapd-core-dependencies): `# even when using g++ for compilation, clangdev dependency is still required`

if I recall correctly it was related to the `rti` option

Reply:
I'll see if I can sneak some time to look more deeply at this later today.

Reply:
But also, don't wait for me :)

Reply:
@scopatz as far as I understand we are building with gcc on linux but we need also clang with rti option. but @pearu can provide info more precise.

Reply:
thanks @scopatz ! :)

Reply:
mapd-core uses clang/llvm both in build and runtime. We should also test if we can build mapd-core without gcc. I'll try it locally..

Reply:
`{{ compiler('cpp') }}` are required by go-cgo..

@scopatz , how one can force `{{ compiler('cpp') }}` to be a clang compiler?